### ISSUE TITLE
Deleted files trigger errors

### ISSUE THREAD

**Issue by Artiom-M** [deajan/osync#203]
With current master and --on-changes option when replication is triggered it is possible that a file has been deleted while Osync scans target, etc. 
This generates errors:

```
TIME: 141 - (ERROR):Could not get file attributes for [/Data/0/3/3/7/0/3.lck].
TIME: 347 - (CRITICAL):Getting file time attributes failed [] on initiator. Stopping execution.
```
Would it be better to check file existence before to try to get attributes? (may be just return code check ?)

**Comment by deajan** [comment]
Could you test latest master perhaps?

**Comment by Artiom-M** [comment]
Seems to be fixed in 1.3.0-rc1

**Comment by deajan** [comment]
Thanks for reporting back.
